frontend/src/config/RpcSettings.ts

```diff
@@ -0,0 +1,19 @@
+import queryString from "query-string";
```

Comment:
Maybe we can use standard `URLSearchParams` instead of an extra npm module

Reply:
I definitely double that:
```typescript
const url = new URL(location.href);
const rpc = url.searchParams.get('rpc');
```

Reply:
Updated: [Here](https://github.com/subspace/subspace-relayer/blob/e2b45e50c286738568136819e1e5081304c70795/frontend/src/config/RpcSettings.ts#L4-L5).